sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/FileTypeSensorTest.java

```diff
@@ -104,8 +111,6 @@ public void whenProjectOutPathsNotPresent_shouldNotAddProjectInfo_shouldNotLog()
   @Test
   public void whenProjectOutPaths_returnsNull_shouldNotAddProjectInfo_shouldNotLog() {
     when(settingsMock.getStringArray("sonar.LANG_KEY.analyzer.projectOutPaths")).thenReturn(null);
-    addFileToFileSystem("foo.language", Type.MAIN);
```

Comment:
I removed these lines because files are not needed to log - if no files are present, it will log FALSE,FALSE as long as the `projectOutPaths` property is set